### PR TITLE
Simpl.html added, the page just has a map with no grid or center square.

### PR DESCRIPTION
**Pull request by mridulnagpal**

### CODE REVIEW

**Comment by mridulnagpal** [comment]
image
Here's a piece of it, also I added two new methods `enableCenterShade()` and `disableCenterShade()` which enable and disable center shade. Adding this to gh-pages as well so that I can link it to README as well

**Comment by mridulnagpal** [comment]
https://mridulnagpal.github.io/leaflet-blurred-location/examples/simple.html

**Comment by jywarren** [comment]
This looks good but if we're inputting by placename, we should make the "in Text" version say the same as the input placename, no? Would we need to have some kind of flag like `location.fromPlacename == true` or something, so we know not to reverse geocode if it was input by string in the first place? 

Or better, could we save the input placename as a string, and return that, as `location.placename`, but if there's no input placename, then we fall back to reverse geocoding? That last sounds like a good solution; let's make a separate issue, and a test and solution in a PR. We can merge this once that's opened.

**Comment by jywarren** [comment]
OK, did that!

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ leaflet-blurred-location provides friendly interfaces for:
 ## Setting up leaflet-blurred-location
 
 To set up the library first clone this repo to your local after that run 'npm install' to install all the neccessary packages required. Then open `examples/index.html` to look at the preview of the library.
+There is a simpler version as well which is a simple location entry namely `examples/simple.html`, you can view all there files online using gh-pages of the repo.
```
Comment (jywarren) [inline README.md]: best simply link to the gh-pages directly as well so people can click and follow the link!

**Pull request merged**